test/test-createTetromino.js

```diff
@@ -0,0 +1,105 @@
+var sinon = require('sinon');
+var assert = require('chai').assert;
+var tetris = require('../tetris');
+describe("Create tetromino", function () {
+    let stub = sinon.stub(tetris,"getRandomTypeOfTetromino")
```

Comment:
Stub is a bad name for this variable. Don't use such generic names - they are not readable

Reply:
Done